app/views/users/index.html.erb

```diff
@@ -1,20 +1,30 @@
-<main>
-<h1>hello</h1>
-  <section>
-    <ul>
-      <% @users.each do |user|%>
-        <%= link_to user do %>
-          <li class="card hover">
-            <div class="user-image border">
-              <%= image_tag(user.photo, class: "photo") %>
-            </div>
-            <div class="user-data border">
-              <h1><%= user.name %></h1>
-              <p class="number-posts">Number of posts: <%= user.post_counter %></p>
-            </div>
-          </li>
-        <% end %>
-      <% end %>
-    </ul>  
-  </section>
-</main>
+<div class="bg-dark min-vh-100" >
+  <div class="container px-2 py-5">
+
+  
+
+  <% @users.each do |user| %>
+  
+  <div class="text-white d-flex gap-3 pb-4" >
+
+    
+    <div class="flex-grow-1  bg-success" >
+       <img src="#" alt=""> 
+    </div>
+    <div class="flex-grow-1 w-50 bg-primary py-4 px-4 d-flex flex-column gap-3" >      
+      
+      <a class="text-white"  href="/users/<%=user.id%>"  >
+        <h2><%=user.name%></h2>      
+      </a>
+      <h6 class="text-end pe-4" >Number of posts: <%=user.posts_counter%>  </h6>
```

Comment:
- [ ] There is an error on your user index page. This should be `post_counter` as indicated by the error message.

  ![image](https://github.com/Marcraphael12/rails-blog-app/assets/36542986/b4a3daf2-ae5c-4720-b601-ed371a940f75)